### DISCUSSION TITLE
Android AAB using default icons

### DISCUSSION THREAD

**Discussion by stuhowarth** [airsdk/Adobe-Runtime-Support#883] [Q&A]
We are using the latest Air SDK (33.1.1.476) and every time we create an Android AAB it is placing default Air icons into some of the folders and these are showing on some of the latest Android devices. Some of the folders we see the default icon includes:

- drawable-xxxhdpi-v4
- drawable-xhdpi-v4
- mipmap-xxhdpi-v4
- mipmap-xhdpi-v4

The icon size appears to be 48x48 but we have that and all the other necessary icons setup correctly in our application.xml. This appears to be the same issue as mentioned in the following threads:

https://github.com/airsdk/Adobe-Runtime-Support/issues/296
https://github.com/airsdk/Adobe-Runtime-Support/issues/36

I have seen in previous release notes that this issue was known and each time it says it will be looked at and in early 2021. I have read the latest release notes and there is a specific mention to AAB files but there was nothing to suggest this problem still exists. Some of these original posts were from 2 years ago so I was wondering if this is still a known issue or if anybody has a fix because I would expect to see a lot more people saying they are having issues than we have found. If anybody can help that would be fab.

We did try unpacking the AAB and manually changing the icons and repacking the AAB but we get certificate issues when we do this so this seems to be a none starter.

**Comment by ajwfrost** [comment]
Hi - we do have an update to the SDK that adds proper support for Android App Bundles, so if you wanted to try this out it would be good to have some real-world testing done on it! The caveat is, you need to have an Android SDK installed as we depend on some of their tooling now...

Can you email me on andrew.frost at harman.com, specifying which platform/edition of the AIR SDK you need, and I'll send you the updated version...

thanks

**Reply by stuhowarth** [reply]
I have emailed you, looking forward to hearing from you :)
